content/blog/grpc-js-1.0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "@gprc/grpc-js 1.0 release announcement"
+date: 2020-04-20
+author:
+  name: Michael Lumish
+  link: https://github.com/murgatroid99
+---
+
+Today we are excited to announce the release of version 1.0 of [@grpc/grpc-js](https://www.npmjs.com/package/@grpc/grpc-js). This library is a reimplementation of the original [grpc](https://www.npmjs.com/package/grpc) library for Node.js, but purely in TypeScript, no longer depending on native C++ code that requires complex compilation and installation procedures.
+
+With this release, the original grpc library is deprecated. We will continue releasing new binary packages for new versions of the Node and Electron runtimes, but it will not receive any new features.
```

Comment:
- "With this release ...": How about something like

  > With release 1.0 of gRPC-JS, we're deprecating the original Node gRPC library.

  **Note:** I'm not seeing a deprecation notice on https://www.npmjs.com/package/grpc

- "We will continue ..."
  > We will continue to release Node gRPC binary packages for new versions of the Node and Electron runtimes, but we will not be adding any new features.